bootstrap-themes/darkly/bootswatch.less

```diff
@@ -129,7 +131,8 @@ table,
 // Forms ======================================================================
 
 input,
-textarea {
+textarea,
+select {
```

Comment:
Is select used outside of forms somewhere else, should it maybe be more specified?